camelConnector/build.gradle

```diff
@@ -61,6 +61,7 @@ dependencies {
     implementation "com.google.code.gson:gson:${gsonVersion}"
     // Used for downloading route documents
     implementation "io.vantiq:vantiq-sdk:${vantiqSDKVersion}"
```

Comment:
Need to load the logging implementation in the main class path.  This gets it loaded into the connector so that logging works as expected.  Otherwise, it's a no-op.